src/components/SortSelector.tsx

```diff
@@ -32,9 +43,11 @@ export const SortSelector = ({ options, sortKey, orderKey, sx = {} }: SortSelect
       sx={{ minWidth: '16rem', ...sx }}
```

Comment:
Foreslår å fjerne dette nå, da den nå tar bredden som trengs for å vise alt innhold uansett. `sx` som prop brukes heller ikke noen plass og kan fjernes
```suggestion
```

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/d860021c-6ff5-40dc-afc2-62d969bd2132)
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/06fd610b-1ab8-4c09-a0a9-d1875644f949)

Reply:
ah denne brukes fortsatt i sentralimport og tasks ja... da kan det ikke fjernes som jeg foreslår nei, med mindre vi setter `sx={{minWidth: "16rem"}}` der det trengs da, som kanskje er mest fornuftig? og fortsatt ha støtte for sx prop

Reply:
forslag:

```
interface SortSelectorProps extends Pick<TextFieldProps, 'sx' | 'variant' | 'size'> {
  orderKey: string;
  options: SortSelectorOption[];
  showLabel?: boolean;
  sortKey: string;
}

const SortSelector = ({
  orderKey,
  options,
  showLabel = true,
  sortKey,
  ...rest
})

<TextField {...rest} (alle andre props) />
```